src/Registry.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * Copyright 2021 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\CloudFunctions;
+
+class Registry
+{
+    public const TYPE_HTTP = "http";
+    public const TYPE_CLOUDEVENT = "cloudevent";
+
+    private static array $fnRegistry = [];
+
+    private function __construct()
+    {
+    }
```

Comment:
For more idiomatic PHP, this should be removed

Reply:
Sounds good!